src/sentry/api/endpoints/project_group_index.py

```diff
@@ -460,6 +471,36 @@ def put(self, request, project):
             id__in=group_ids,
         )
 
+        discard = result.get('discard')
+        if discard:
+            if not features.has('projects:custom-filters', project, actor=request.user):
+                return Response({'detail': ['You do not have that feature enabled']}, status=400)
+            group_list = list(queryset)
+            for group in group_list:
+                with transaction.atomic():
+                    try:
+                        tombstone = GroupTombstone.objects.create(
+                            previous_group_id=group.id,
+                            project_id=group.project_id,
+                            level=group.level,
+                            message=group.message,
+                            culprit=group.culprit,
+                            type=group.get_event_type(),
```

Comment:
hmm, so the whole method is:
```
def get_event_type(self):
    return self.data.get('type', 'default')
```
do you mean save the whole `data` field? or just the `type` part of the data field? i'm fine with whatever, mostly just did it this way because i thought we were trying to save as little data as possible.